lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -76,7 +76,7 @@ def attr_encrypted(*attrs)
               # attributes are handled, @attributes[attr].value is nil which
               # breaks attribute_was. Setting it here returns us to the expected
               # behavior.
-              if Gem::Requirement.new('>= 5.2').satisfied_by?(RAILS_VERSION)
+              if RAILS_VERSION < Gem::Version.new(6.0) && RAILS_VERSION >= Gem::Version.new(5.2)
```

Comment:
This is the key to compatibility with our repo.

Reply:
Do we need this block? We're on rails 6+ no?

Reply:
That's true, we could pull it out entirely. The only important thing for us is that `          set_attribute_was(attr,__send__(attr))` is not called. This was just the fix suggested upstream that I copied over. Could delete the entire thing if there's a preference for that.